src/services/search/listingSearch.ts

```diff
@@ -92,6 +153,14 @@ const searchForListings = ({
   return postData({ path, body, options: otherOptions })
 }
 
+const sanitizeListing = ({
```

Comment:
Whats is the function sanitizing? It looks more like converting some data based on the function called...
Btw why is the `decodeDate` function adding a month to the passed date this looks very strange to me...

Reply:
It ensures that the date format is as expected (with a decoded date object). Kept the naming in line with the existing functions.
JS indexes months from `0` so yes, we do add one to have an indexing a sane person would expect when looking at a date object.

Reply:
Ah I see, js and dates 🙈